src/components/layout/Undernames/Undernames.tsx

```diff
@@ -0,0 +1,191 @@
+  const [tableColumns, setTableColumns] =
+    useState<ColumnType<UndernameMetadata>[]>();
+  const [tablePage, setTablePage] = useState<number>(1);
+
+  useEffect(() => {
+    if (!id) {
+      eventEmitter.emit('error', new Error('Missing PDNT transaction ID.'));
+      navigate('/manage/pdnts');
+      return;
+    }
+    setPDNTId(new ArweaveTransactionID(id));
+    arweaveDataProvider
+      .getContractState<PDNTContractJSON>(new ArweaveTransactionID(id))
+      .then((state) => setPDNTState(state));
+  }, [id]);
+
+  useEffect(() => {
+    if (!id) {
+      eventEmitter.emit('error', new Error('Missing PDNT transaction ID.'));
+      navigate('/manage/pdnts');
+      return;
+    }
+    if (isArweaveTransactionID(id)) {
+      setTableLoading(undernameTableLoading);
+      setTableData(undernameRows);
+      setTableColumns(undernameColumns);
+      setPercentLoaded(percentUndernamesLoaded);
+      setSelectedRow(selectedUndernameRow);
+      const baseIndex = Math.max((tablePage - 1) * 10, 0);
+      const endIndex = tablePage * 10;
+      const filteredData = undernameRows.slice(baseIndex, endIndex);
+      setFilteredTableData(filteredData);
+    }
+  }, [
+    id,
+    undernameSortAscending,
+    undernameSortField,
+    undernameRows,
+    selectedUndernameRow,
+    undernameTableLoading,
+    percentUndernamesLoaded,
+  ]);
+
+  function updatePage(page: number) {
+    setTablePage(page);
+  }
+
+  return (
+    <>
+      <div className="page">
+        <div className="flex-column">
+          <div className="flex flex-justify-between">
+            <div className="flex flex-row text-medium white bold">
+              <span className="flex faded text-medium bold">
+                <button
+                  className="faded text-medium bold underline link center"
+                  onClick={() => navigate('/manage/pdnts')}
+                >
+                  <ArrowLeft
+                    width={30}
+                    height={20}
+                    viewBox={'0 0 20 20'}
+                    fill={'var(--text-white)'}
+                  />
+                  Manage PDNTs
+                </button>
+                &nbsp;/&nbsp;
+                <Tooltip
+                  placement="top"
+                  title={id}
+                  showArrow={true}
+                  overlayStyle={{
+                    maxWidth: 'fit-content',
+                  }}
+                >
+                  <button
+                    className="faded text-medium bold underline link center"
+                    onClick={() =>
+                      navigate(`/manage/pdnts/${pdntId?.toString()}`)
+                    }
+                  >
+                    {pdntState?.name.length ? pdntState.name : '[PDNT]'}
+                  </button>
+                </Tooltip>
+                &nbsp;/&nbsp;
+                <span className="text-medium white">Manage Undernames</span>
+              </span>
+            </div>
```

Comment:
the size of the text of these is smaller than the `Manage ANT's` page

Reply:
fixed